build.gradle

```diff
@@ -0,0 +1,238 @@
+plugins {
+    id 'com.github.johnrengelman.shadow' version '4.0.4'
+    id 'java'
+    id 'maven'
+    id 'application'
+    id 'edu.sc.seis.launch4j' version '2.5.1'
+    // id 'edu.sc.seis.macAppBundle' version '2.3.0'
+    id "nebula.ospackage" version "8.2.0"
+    id 'org.gradle.crypto.checksum' version '1.2.0'
+    id 'com.palantir.git-version' version '0.12.2'
+    id 'io.spring.dependency-management' version '1.0.1.RELEASE' //for logging - log4j
+	
+	  id 'nu.studer.jooq' version '3.0.3'
+	  
+	  id "org.flywaydb.flyway" version "8.2.0"
+	
+    id "de.undercouch.download" version "1.2"
+    
+    id 'jacoco'
+}
+
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+// In this section you declare where to find the dependencies of your project
+repositories {
+    // Use jcenter for resolving your dependencies.
+    // You can declare any Maven/Ivy/file repository here.
+    jcenter()
+    maven { url 'https://jitpack.io' }
+}
+
+dependencies {
+    compile 'com.github.signum-network:signumj:v0.20.0'
+    compile 'io.reactivex:rxjava:1.3.8'
+    compile 'org.bouncycastle:bcprov-jdk15on:1.69'
+    compile 'org.ehcache:ehcache:3.9.9'
+    compile 'org.flywaydb:flyway-core:8.2.0'
+    compile 'com.google.code.gson:gson:2.8.5'
+    compile 'com.h2database:h2:1.4.199'
+    compile 'commons-cli:commons-cli:1.4'
+    compile 'com.zaxxer:HikariCP:3.3.1'
+    compile 'org.jocl:jocl:2.0.1'
+    compile 'org.jooq:jooq:3.14.8'
+    compile 'org.mariadb.jdbc:mariadb-java-client:2.4.3'
+    compile 'org.owasp.encoder:encoder:1.2.2'
+    compile 'org.bitlet:weupnp:0.1.4'
+    
+    compile 'com.google.zxing:core:3.4.0'
+    compile 'com.google.zxing:javase:3.4.0'
+    
+    compile 'org.slf4j:slf4j-api:1.7.28'
+    compile 'org.slf4j:slf4j-jdk14:1.7.28'
+    
+    compile 'org.eclipse.jetty:jetty-server:9.4.43.v20210629'
+    compile 'org.eclipse.jetty:jetty-servlet:9.4.43.v20210629'
+    compile 'org.eclipse.jetty:jetty-servlets:9.4.43.v20210629'
+    compile 'org.eclipse.jetty:jetty-rewrite:9.4.43.v20210629'
+    
+    compile 'javax.annotation:javax.annotation-api:1.3.2'
+    
+    compile 'org.apache.httpcomponents:httpclient:4.5.13'
+
+    compile 'com.github.jiconfont:jiconfont:1.0.0'
+    compile 'com.github.jiconfont:jiconfont-swing:1.0.1'
+    compile 'com.github.jiconfont:jiconfont-font_awesome:4.7.0.1'
+
+    //logging
+    compile 'org.apache.logging.log4j:log4j-api:2.16.0'
```

Comment:
!!! good - people will look into this.. but we should take the latest: 2.17.1
https://logging.apache.org/log4j/2.x/

Version changed a lot recently due to the sec issue

Reply:
Updated to 2.17.1